packages/coinstac-ui/app/render/components/app.js

```diff
@@ -1,5 +1,6 @@
 import Notify from './notification';
-import React, { Component, PropTypes } from 'react';
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
```

Comment:
The separate `prop-types` package is new in React 15.5, heralded by a console warning. I imagine they'll deprecate in 16.x.